src/modules/convex/utils_regularization.cpp

```diff
@@ -79,8 +79,13 @@ class ScalesState
 AnyType utils_var_scales_transition::run (AnyType& args)
 {
     ScalesState<MutableArrayHandle<double> > state = args[0];
+    try {
+        args[1].getAs<MappedColumnVector>();
+    } catch(const ArrayWithNullException &e) {
+      //  warning("Input array contains NULL values, skipping it for mean and std computations");
+        return state;
+    }
```

Comment:
A more efficient way is to declare an empty handle outside the try block and rebind the memory later:

```c++
MappedColumnVector x(NULL);
try {
    new (&x) MappedColumnVector(args[1].getAs<MappedColumnVector>());
    // code ...
}
```

Save goes for other places where this applies.